terracotta/server/flask_api.py

```diff
@@ -110,6 +110,10 @@ def create_app(debug: bool = False, profile: bool = False) -> Flask:
 
     if profile:
         from werkzeug.contrib.profiler import ProfilerMiddleware
-        new_app.wsgi_app = ProfilerMiddleware(new_app.wsgi_app, restrictions=[30])
+        setattr(
+            new_app,
+            'wsgi_app',
+            ProfilerMiddleware(new_app.wsgi_app, restrictions=[30]
+        )
```

Comment:
??

Reply:
I guess because this also works if `new_app.wsgi_app` doesn't exist already.

Reply:
No acutally, it won't. So I would also like to know 😄

Reply:
It's the recommended workaround for https://github.com/python/mypy/issues/2427